com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ArchiveReader.java

```diff
@@ -102,28 +90,27 @@ private void readArchiveData() throws IOException {
         // Once the hashes are read, the rest of the entries are read.
 
         ManifestData manifestData = null;
+        HashMap<ByteBuffer, ResourceEntry> temp_map = new HashMap<ByteBuffer, ResourceEntry>();
```

Comment:
Nice improvement! Given our naming conventions I think it should be named `tempMap`. Perhaps also better to name it for what it is, like `hashToResourceLookup`?